src/Interface/inc/ICalculator.h

```diff
@@ -8,8 +8,11 @@
  */
 /////////////////////////////////////////////////////
 
+#include <iostream>
 #include <string>
 
+#include <CalculatorStrings.h>
```

Comment:
Only use angle brackets <> for system libraries. All others should use quotes ""